### DISCUSSION TITLE
Video/Image source link

### DISCUSSION THREAD

**Discussion by volt001** [Docile-Alligator/Infinity-For-Reddit#964] [Ideas]
I love the app but would appreciate it if there was a link to the  imgur/gfycat/redgifs like how there is one on the main reddit app
![Screenshot_20220220-151338_Reddit](https://user-images.githubusercontent.com/65422521/154868630-341f74b7-9750-41f3-936d-d40db3f0b4d2.jpg)
.

**Comment by Docile-Alligator** [comment]
Good idea!

**Reply by Maio-msdos** [reply]
Could we also include a way to filter posts based on the video source e.g. exclude all video posts from gfycat etc :)

**Reply by Wladefant** [reply]
@Maio-msdos please add each feature request to its own post
